model_api/cpp/adapters/src/openvino_adapter.cpp

```diff
@@ -34,23 +44,48 @@ void OpenVINOInferenceAdapter::loadModel(const std::shared_ptr<const ov::Model>&
 }
 
 InferenceOutput OpenVINOInferenceAdapter::infer(const InferenceInput& input) {
+    auto request = (*asyncQueue)[asyncQueue->get_idle_request_id()];
```

Comment:
```suggestion
    auto request = asyncQueue,get()[asyncQueue->get_idle_request_id()];
```
It was hard to read

Reply:
Changed to `asyncQueue->operator[]()`, because after `.get()` dereferencing is required as well

Reply:
I thought I deleted that comment :)